From 0498ec4afceebc5fed294db5c79d3d167b037a2b Mon Sep 17 00:00:00 2001
From: Vasily Khoruzhick <anarsoul@gmail.com>
Date: Wed, 8 Jan 2020 21:10:33 -0800
Subject: [PATCH 43/51] lima: drop support for R8G8B8 format

We can only sample from 24-bit packed format and can't render into it and
it causes chromium-based browsers to fail when they create FBO with GL_RGB
format. Drop R8G8B8 alltogether so mesa can promote it to RGBX format.

Reviewed-by: Qiang Yu <yuq825@gmail.com>
Signed-off-by: Vasily Khoruzhick <anarsoul@gmail.com>
---
 src/gallium/drivers/lima/lima_format.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gallium/drivers/lima/lima_format.c b/src/gallium/drivers/lima/lima_format.c
index a599ee67eec..f7e4d66e148 100644
--- a/src/gallium/drivers/lima/lima_format.c
+++ b/src/gallium/drivers/lima/lima_format.c
@@ -69,7 +69,6 @@ static const struct lima_format lima_format_table[] = {
    LIMA_FORMAT(B8G8R8A8_SRGB,      RGBA_8888, B8G8R8A8, false),
    LIMA_FORMAT(R8G8B8X8_UNORM,     RGBX_8888, B8G8R8A8, true),
    LIMA_FORMAT(B8G8R8X8_UNORM,     RGBX_8888, B8G8R8A8, false),
-   LIMA_FORMAT(R8G8B8_UNORM,       RGB_888,   NONE,     true),
    LIMA_FORMAT(B5G6R5_UNORM,       BGR_565,   B5G6R5,   false),
    LIMA_FORMAT(Z24_UNORM_S8_UINT,  Z24S8,     Z24S8,    false),
    LIMA_FORMAT(Z24X8_UNORM,        Z24S8,     Z24S8,    false),
-- 
2.23.0

